src/main/java/com/baidu/hugegraph/formatter/KGDumper/DumpKGFormatter.java

```
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2017 HugeGraph Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.baidu.hugegraph.formatter.kgdumper;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import com.baidu.hugegraph.base.Printer;
+import com.baidu.hugegraph.formatter.Formatter;
+import com.baidu.hugegraph.structure.JsonGraph.JsonVertex;
+import com.baidu.hugegraph.structure.JsonGraph.JsonEdge;
+
+public class DumpKGFormatter implements Formatter {
+
+    private ComputeSign cs;
+
+    public DumpKGFormatter() {
+        try {
+            cs = new ComputeSign("./trade_value", "GBK");
+        } catch (IOException e) {
+            e.printStackTrace();
```

don't swallow exception